src/tests/functional/plugin/conformance/subgraphs_dumper/src/ops_cache.cpp

```diff
@@ -196,10 +196,10 @@ OPCache::serialize_function(const std::pair<std::shared_ptr<ov::Node>, LayerTest
         }
         std::replace(op_name.begin(), op_name.end(), '/', '_');
         std::replace(op_name.begin(), op_name.end(), '\\', '_');
-        // TODO: Possible names collision
-        auto xml_path = current_op_folder + CommonTestUtils::FileSeparator + op_name + ".xml";
-        auto bin_path = current_op_folder + CommonTestUtils::FileSeparator + op_name + ".bin";
-        auto meta_info = current_op_folder + CommonTestUtils::FileSeparator + op_name + ".meta";
+        auto filePrefix = CommonTestUtils::generateTestFilePrefix();
```

Comment:
@olpipi Why do you made such changes??? It causes of the crashes in `subgraphdumper`.
Next time ask me than make such changes
FYI @ilyachur